src/pages/workspace/companyCards/BankConnection/index.native.tsx

```diff
@@ -169,7 +180,7 @@ function BankConnection({policyID: policyIDFromProps, feed, route}: BankConnecti
                 onBackButtonPress={handleBackButtonPress}
             />
             <FullPageOfflineBlockingView addBottomSafeAreaPadding>
-                {!!url && !isConnectionCompleted && !isPlaid && !isNewFeedHasError && (
+                {!!url && !isConnectionCompleted && !isPlaid && !isNewFeedHasError && !isBlockedToAddNewFeeds && !isAllFeedsResultLoading && (
```

Comment:
`isBlockedToAddNewFeeds` is only available until all feeds have loaded

```suggestion
                {!!url && !isConnectionCompleted && !isPlaid && !isNewFeedHasError && !isAllFeedsResultLoading && !isBlockedToAddNewFeeds && (
```